modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/values/OperationValuesTestCase.java

```diff
@@ -144,4 +144,22 @@ public void userErrorWhenResolvingValues() throws Exception {
     assertThat(resolvingFailure.getFailureCode(), is("CUSTOM_ERROR"));
     assertThat(resolvingFailure.getMessage(), is(ERROR_MESSAGE));
   }
+
+  @Test
+  public void singleValuesEnabledParameterWithMoreThanOneFieldValues() throws Exception {
```

Comment:
In the cases of the feature added in this PR, I think it would be good to also assert the partName of the values.

Reply:
I think this comment was not addressed.

Reply:
It is